src/index.html

```diff
@@ -13,6 +13,40 @@
     />
```

Comment:
Link to a stylesheet with a .scss extension won't work in a browser. You should compile the SCSS to CSS and link to the .css file instead.